src/environments/environment.ts

```diff
@@ -33,7 +33,7 @@ export const environment = {
   remoteLoggingDsn: undefined, // only set for production mode in environment.prod.ts
 
   demo_mode: true,
-  session_type: SessionType.mock,
+  session_type: SessionType.local,
```

Comment:
before checking into github, revert this (we don't want to merge the SessionType change into our master branch)